src/components/avatar/index.tsx

```diff
@@ -34,6 +34,10 @@ export const Avatar: React.FC<AvatarProps> = ({ className, src, size = 54, alt,
   const fontSize = getFontSize(theme, size);
   const [isImageLoadedError, setImageLoadedError] = React.useState(false);
 
+  React.useEffect(() => {
+    setImageLoadedError(false);
+  }, [src]);
```

Comment:
Do you need to check if `isImageLoadedError` is true?

Reply:
@vlacher12 
Maybe upload errors should be handled outside of this component. The component can have an onError prop

Reply:
@vozbrann @vlacher12 so, what is better decision?

Reply:
what's the purpose of  `onError={() => setImageLoadedError(true)}`?

Reply:
I think that this component should not manage the error state. An error and src should be passed in this component as a prop

Reply:
Agree